_component_design/tabs.md

```diff
@@ -7,20 +7,128 @@ usage: Teaser Text
 preview-image: components/preview-images/tabs.svg
 resource: true
 status: unknown
-last-modified: 2017-08-17
+last-modified: 2017-09-11
 ---
 
 {% include toc.html %}
 
-Tabs are used to separate content into different views, which can be displayed
-one at a time without a page refresh. Tabs provide easy access to additional
-content without requiring the user to either navigate to a new page or scroll
-too much. The default view should always display the left-most tab.
+Use tabs to separate content into different views that can each be
+displayed quickly without a page refresh. Tabs provide easy access to
+a set of content without requiring the user to either move to a new
```

Comment:
sets of content* ? Seems to switch tenses. Or I could just be tired.

Reply:
changed